app/javascript/components/maps/map/components/draw/index.js

```diff
@@ -59,6 +61,7 @@ class MapDraw extends PureComponent {
 
   closeDrawing = () => {
     const { map } = this.props;
+    track('analysisDrawCancel');
```

Comment:
Only thought here is that this will happen when a draw is completed due to closing the shape and not only when you cancel. Maybe this isnt a big problem, as we can compare to real 'drawComplete' events and see the difference.

Reply:
Removed, it's already implemented [here](https://github.com/Vizzuality/gfw/blob/d8baa147b66268aadcd4658561ace0ac94ea89a3/app/javascript/components/maps/components/analysis/components/chose-analysis/component.jsx#L103).